### ISSUE TITLE
Cardinal Adina log fails to parse

### ISSUE THREAD

**Issue by Potawto** [baaron4/GW2-Elite-Insights-Parser#593]
Cardinal Adina log fails to parse with the error below

```
Creating File(s)
Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
```

[20210926-021833.log](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/7231676/20210926-021833.log)
[20210926-021833.zevtc.zip](https://github.com/baaron4/GW2-Elite-Insights-Parser/files/7231680/20210926-021833.zevtc.zip)

**Comment by EliphasNUIT** [comment]
You are running an outdated version of arcdps where cast events are completely missing. I intend to add sanity checks so that such logs still parse (without phases ofc) but please update your arcdps.

**Comment by Potawto** [comment]
How recent is that arcdps change? Is it very recent?

I could have sworn I was running the most recent version of arcdps, as I had been constantly redownloading it since DX11 beta started. Another guild member failed to parse his own log for the same fight as well, though I don't know what version he was running.

**Comment by EliphasNUIT** [comment]
You are on one of the "EVTC20210923" version, there were multiple builds that day. The very latest "EVTC20210923" fixes the absence of cast events. Most recent is "EVTC20210926".

**Comment by Potawto** [comment]
I see, thank you <3

Apologies for the issue post!

**Comment by EliphasNUIT** [comment]
Your issue is still valid, the log is not broken to the point of not being able to be parsed at all. In next, this log will parse, albeit with phases missing.

**Comment by EliphasNUIT** [comment]
Fixed in 2.38.0.0.
